public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<head>
+  <h2>FEC</h2>
+</head>
+<body>
+  <div>
+    <div id='root'></div>
+    <script src='bundle.js'></script>
```

Comment:
The script tag should be at the end of the body and outside of any inner body elements.